# Biological Data Scientist
Data scientist and molecular biologist with 14 years of research experience in microbiology, synthetic biology, and cell biology.

## Work Experience
**Senior Scientist I through II at Gritstone bio (_June 2021 - Present_)**
- Led the pre-IND work for the multi-respiratory virus vaccine project as principal scientist and project manager allowing selection of a clinical candidate vaccine
- Created and implemented a _de novo_ next-generation sequencing analysis computational pipeline for plasmid sequence verification, improving processing time over manual methods from hours to minutes

**Scientist I through II at Zymergen (_October 2018 - June 2021_)**
- Utilized computational methods to design combinatorial, multi-gene constructs _in silico_ and oversaw the construction of more than 3,000 engineered bacterial strains for increased biosynthetic product yield
- Performed phylogenetic analyses and molecular design for the expression of more than 250 novel biosynthetic gene clusters to build Zymergen’s natural product portfolio
- Drove technical updates and project steering meetings with a high-profile external collaborator

**Post-doctoral Research Fellow at the CDC (_October 2016 - October 2018_)**
- Leveraged computational methods to design a library of 9447 distinct viral peptides for a novel serology assay to detect emerging viruses
- Elucidated key molecular mechanisms underlying strain-specific pathogenicity of Crimean-Congo hemorrhagic fever virus

**Graduate Research Associate at MIT (_June 2011 - October 2016_)**
- Generated genome-wide DNA copy number and transcriptome profiles in five distinct Drosophila tissues and analyzed datasets using sequence alignment pipelines on high-performance computing clusters
- Uncovered novel mechanisms regulating differential replication in Drosophila development

## Projects

**Automated tracking of SARS-CoV-2 variant predominance- Gritstone bio (_March 2024 - Present_)**
- Utilized the CDC SARS-CoV-2 variant database API to pull variant frequency data at regular intervals
- Cleaned and transformed data using pandas to determine changes in predominant variants in the USA
- Working to deploy tool to the cloud that alerts users of new variants, their frequencies of detection, and their associated sequence records

**Genome-wide analysis of RNA secondary structure and mammalian gene expression - Gritstone bio (_June 2023 - July 2023_)**
- Utilized Biopython to parse human genome sequence and extract defined regions around the start codon transcriptome-wide
- Created a bash pipeline to assess RNA secondary structure of all human genes using RNAfold
- Parsed and aggregated proteomics datasets from HEK293 cells with gene sequences using pandas
- Established correlation of strong RNA secondary structure at transcription start site region with increased protein expression

**Identification of highly-conserved epitopes in viral sequences (_June 2022 - December 2023_)**
- Leveraged Python to assess conservation at each amino acid position in a multiple-sequence alignment
- Extracted candidate 8-, 9-, 10-, and 11-mer epitope sequences with high levels of conservation for T-cell based therapeutic design

## Skills
- **Languages**: Python, SQL
- **Data science pipeline**: Querying, cleaning, aggregating, data visualization, descriptive statistics, hypothesis testing
  - Python - pandas, numpy, matplotlib, seaborn, plotly, scikit-learn, scipy, statsmodels, biopython, beautifulsoup4
  - MySQL, BigQuery
- **Project management**: Project scoping, technical data presentation, management of external collaborators/ CROs, management of direct reports

## Education
- Ph.D., Biology | MIT (_Oct 2016_)								       		
- B.S., Molecular and Cellular Biology | UC Davis (_June 2010_)
  
## Publications

1. **Hua, B. L.**, Scholte, F. E., Ohlendorf, V., Kopp, A., Marklewitz, M., Drosten, C., Nichol, S. T., Spiropoulou, C., Junglen, S., & Bergeron, É. (2020). A single mutation in Crimean-Congo hemorrhagic fever virus discovered in ticks impairs infectivity in human cells. eLife, 9, e50999. [Link](https://doi.org/10.7554/eLife.50999)
2. Scholte, F. E. M., **Hua, B. L.**, Spengler, J. R., Dzimianski, J. V., Coleman-McCray, J. D., Welch, S. R., McMullan, L. K., Nichol, S. T., Pegan, S. D., Spiropoulou, C. F., & Bergeron, É. (2019). Stable Occupancy of the Crimean-Congo Hemorrhagic Fever Virus-Encoded Deubiquitinase Blocks Viral Infection. mBio, 10(4), e01065-19. [Link](https://doi.org/10.1128/mBio.01065-19)
3. **Hua, B. L.**, Bell, G. W., Kashevsky, H., Von Stetina, J. R., & Orr-Weaver, T. L. (2018). Dynamic changes in ORC localization and replication fork progression during tissue differentiation. _BMC Genomics_, 19(1), 623. [Link](https://doi.org/10.1186/s12864-018-4992-3)
4. **Hua, B. L.**, & Orr-Weaver, T. L. (2017). DNA Replication Control During Drosophila Development: Insights into the Onset of S Phase, Replication Initiation, and Fork Progression. Genetics, 207(1), 29–47. [Link](https://doi.org/10.1534/genetics.115.186627)
5. Habrian, C., Chandrasekhara, A., Shahrvini, B., **Hua, B.**, Lee, J., Jesinghaus, R., Barry, R., Gitai, Z., Kollman, J., & Baldwin, E. P. (2016). Inhibition of Escherichia coli CTP Synthetase by NADH and Other Nicotinamides and Their Mutual Interactions with CTP and GTP. Biochemistry, 55(39), 5554–5565. [Link](https://doi.org/10.1021/acs.biochem.6b00383)
6. **Hua, B. L.**, Li, S., & Orr-Weaver, T. L. (2014). The role of transcription in the activation of a Drosophila amplification origin. G3 (Bethesda, Md.), 4(12), 2403–2408. [Link](https://doi.org/10.1534/g3.114.014050)

## Achievements

- Data Scientist with Python Track completion, Data Camp, 2023
- American Society for Virology Postdoctoral Scholar Travel Award, 2018
- Journal of Virology Best Poster Award, Negative Strand Virus Meeting, 2018
- American Society for Microbiology/CDC Postdoctoral Research Fellowship, 2016
- MIT School of Science Graduate Student Fellowship in Cancer Research, MIT, 2015
